leaderboard/enriching/enricher.go

```diff
@@ -127,21 +214,3 @@ func membersModelToProto(leaderboardID string, members []*model.Member) []*podiu
 
 	return protoMembers
 }
-
-func protoToMemberModels(protoMembers []*podium_leaderboard_webhooks_v1.Member) []*model.Member {
```

Comment:
I do not have much context, but this seems to be important. Is there a reason for it to be ripped out?

Reply:
I changed it to just get the metadata from the proto. No need to have this mapping anymore.